src/components/CommunityPostComponent/CommunityPostComponent.module.scss

```diff
@@ -1,4 +1,26 @@
 .post {
   width: 100%;
   box-shadow: 0px 2px 10px lightgray;
+  font-family: "Roboto", sans-serif;
```

Comment:
we've been using Ubuntu in the photoshop files, should we stick with it or move everything over to Roboto?